data/blog/electron-10-0.md

```diff
@@ -74,5 +74,5 @@ For information on planned breaking changes in upcoming versions of Electron, [s
 ### Continued Work for Deprecation of `remote` Module (in Electron 11)
 We started work to remove the remote module in [Electron 9](https://www.electronjs.org/blog/electron-9-0) and we're continuing plans to remove the `remote` module. In Electron 11, we plan to continue refactor work for implementing [WeakRef](https://v8.dev/features/weak-references) as we have done in Electron 10. Please read and follow [this issue](https://github.com/electron/electron/issues/21408) for full plans and details for deprecation.
 
-### Final Step for Requiring Native Node Modules to be Context Aware or N-API (in Electron 11)
-From Electron 6 onwards, we've been laying the groundwork to require [native Node modules](https://nodejs.org/api/addons.html) loaded in the renderer process to be either [N-API](https://nodejs.org/api/n-api.html) or [Context Aware](https://nodejs.org/api/addons.html#addons_context_aware_addons). Enforcing this change allows for stronger security, faster performance, and reduced maintenance workload. The final step of this plan is to remove the ability to disable render process reuse in Electron 11. Read [this issue](https://github.com/electron/electron/issues/18397) for full details including the proposed timeline.
+### Final Step for Requiring Native Node Modules to be Context Aware or N-API (in Electron 12)
```

Comment:
Can we add a note that this was changed because of the quiet release?

Reply:
@erickzhao Definitely! Will add that language in - I might change "quiet release", since we did technically have breaking changes, but I'll highlight that we moved it to make it easier for devs to upgrade to Electron 11 and start using the new binary that supports Apple's M1 if that sounds good?

Reply:
Yep, makes sense to me. As long as there's an edit note so that people don't refer back to the Electron 10 blog post and are surprised that the version number is different from what they remembered.